tests/scripts/multi-node/build-rook.sh

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
```

Comment:
do you think we should rename this script to `deploy-rook.sh` to give a better indication that it will actually deploy Rook to your cluster?

Reply:
The thing is the script does many things, so yes the current name is not 100% accurate.